<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainControlsToolbar</class>
 <widget class="QToolBar" name="MainControlsToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>48</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Main Controls</string>
  </property>
  <property name="windowIcon">
   <iconset resource="scanner.qrc">
    <normaloff>:/scanner/Resources/Icons/3D.jpg</normaloff>:/scanner/Resources/Icons/3D.jpg</iconset>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
  <property name="floatable">
   <bool>true</bool>
  </property>
  <action name="actionOpenData">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/pqOpen24.png</normaloff>:/scanner/Resources/Icons/pqOpen24.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="statusTip">
    <string>Open</string>
   </property>
   <property name="PV_MUST_BE_MASTER_TO_SHOW" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/pqSave24.png</normaloff>:/scanner/Resources/Icons/pqSave24.png</iconset>
   </property>
   <property name="text">
    <string>Save Data...</string>
   </property>
   <property name="PV_MUST_BE_MASTER_TO_SHOW" stdset="0">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/pqHelp24.png</normaloff>:/scanner/Resources/Icons/pqHelp24.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/pqUndo24.png</normaloff>:/scanner/Resources/Icons/pqUndo24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="iconText">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo(Ctrl+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/pqRedo24.png</normaloff>:/scanner/Resources/Icons/pqRedo24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo(Ctrl+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLoadImages">
   <property name="icon">
    <iconset resource="scanner.qrc">
     <normaloff>:/scanner/Resources/Icons/importPictureDocument.ico</normaloff>:/scanner/Resources/Icons/importPictureDocument.ico</iconset>
   </property>
   <property name="text">
    <string>LoadImages</string>
   </property>
  </action>
  <addaction name="actionLoadImages"/>
  <addaction name="actionOpenData"/>
  <addaction name="actionSaveData"/>
  <addaction name="separator"/>
  <addaction name="actionUndo"/>
  <addaction name="actionRedo"/>
  <addaction name="separator"/>
  <addaction name="actionHelp"/>
 </widget>
 <resources>
  <include location="scanner.qrc"/>
 </resources>
 <connections/>
</ui>
